.travis.yml

```diff
@@ -2,28 +2,26 @@
 # * http://www.objc.io/issue-6/travis-ci.html
 # * https://github.com/supermarin/xcpretty#usage
 
-osx_image: xcode9.2
+osx_image: xcode9.3
```

Comment:
We should look at starting to migrate to 9.4 images where possible/needed

Reply:
I attempted to do so but it didn't work for 9.4. could be they haven't set it up yet. is this the source of truth?  https://docs.travis-ci.com/user/reference/osx/

Reply:
As far as I know, yes